pycheribuild/projects/cross/x11.py

```diff
@@ -79,6 +79,14 @@ def setup(self):
             # Needed for many of the projects but not all of them:
             self.configure_args.append("--enable-malloc0returnsnull")
 
+    @property
+    def default_ldflags(self):
+        result = super().default_ldflags
+        if not self.compiling_for_host():
+            result.append(DoNoQuoteStr("-Wl,--enable-new-dtags,-rpath,'$$ORIGIN/../lib'"))
+            result.append("-Wl,-z,origin")
```

Comment:
Probably shouldn't have copy-pasted that flag, since it means rtld won't look in the normal search dirs.